tests/Postman/collection.json

```diff
@@ -1,3214 +1,3492 @@
-                  "value": "{\"coupon_id\":\"15\"}",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][gender]",
-                  "value": "f",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][first_name]",
-                  "value": "Krystel",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][last_name]",
-                  "value": "Mbarga",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][birthday]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][company]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][street_1]",
-                  "value": "7 cours de larche guedon",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][street_2]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][city]",
-                  "value": "Torcy",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][zipcode]",
-                  "value": "77200",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][country]",
-                  "value": "FR",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[invoice_address][state]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][gender]",
-                  "value": "f",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][first_name]",
-                  "value": "Krystel",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][last_name]",
-                  "value": "Mbarga",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][birthday]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][company]",
-                  "value": "",
-                  "type": "text"
-                },
-                {
-                  "key": "cart[delivery_address][street_1]",
-                  "value": "7 cours de larche g
```

Comment:
i may not be understanding thing ,but what we testing here?

Reply:
I guess it was about testing the h20 coupon?!

Reply:
ugh, github messed up the lines due to whitespace ignoring I had to use. I was pointing to the check_coupon that you do
```
var jsonData = pm.response.json();
if (jsonData.hasOwnProperty('items') && jsonData.items.length > 0) {
    pm.environment.set("order_item_id", jsonData.items[0].item_id);
}
```